Src/WitsmlExplorer.Frontend/models/jobs/messageObjectReference.ts

```diff
@@ -0,0 +1,4 @@
+export default interface MessageObjectReference {
+  wellUid: string;
+  wellboreUid: string;
```

Comment:
Should include messageUid as part of MessageObjectReference

Reply:
Thanks! 😃  Fixed